src/services_models/services_constructor.jl

```diff
@@ -67,7 +67,7 @@ function construct_service!(
         end
 
         #Variables
-        activeservice_variables!(psi_container, service, contributing_devices)
+        add_variables!(ServiceVariable, psi_container, service, contributing_devices)
```

Comment:
we need a more specific name here since not all the service variables are the same name. This will ve a reserve variable.